core/src/main/scala/cats/Foldable.scala

```diff
@@ -578,42 +578,60 @@ trait Foldable[F[_]] extends UnorderedFoldable[F] with FoldableNFunctions[F] { s
    * scala> import cats.syntax.all._
    * scala> def parseInt(s: String): Option[Int] = Either.catchOnly[NumberFormatException](s.toInt).toOption
    * scala> val F = Foldable[List]
-   * scala> F.traverse_(List("333", "444"))(parseInt)
+   * scala> F.traverseVoid(List("333", "444"))(parseInt)
    * res0: Option[Unit] = Some(())
-   * scala> F.traverse_(List("333", "zzz"))(parseInt)
+   * scala> F.traverseVoid(List("333", "zzz"))(parseInt)
    * res1: Option[Unit] = None
    * }}}
    *
    * This method is primarily useful when `G[_]` represents an action
    * or effect, and the specific `A` aspect of `G[A]` is not otherwise
    * needed.
    */
-  def traverse_[G[_], A, B](fa: F[A])(f: A => G[B])(implicit G: Applicative[G]): G[Unit] =
+  def traverseVoid[G[_], A, B](fa: F[A])(f: A => G[B])(implicit G: Applicative[G]): G[Unit] =
     foldRight(fa, Always(G.unit)) { (a, acc) =>
       G.map2Eval(f(a), acc) { (_, _) =>
         ()
       }
     }.value
 
+  /**
+   * Alias for `traverseVoid`.
+   *
+   * @deprecated this method should be considered as deprecated;
+   *             refrain from using this method and consider switching to `traverseVoid`.
```

Comment:
I wouldn't say `consider switching` I would say `replaced by traverseVoid`.

Reply:
@johnynek , thank you for the comment. Before I start making changes, I'd like to make sure we are on the same page. Are you suggesting to rephrase the notice this way (full sentence):
```
@deprecated this method should be considered as deprecated and replaced by `traverseVoid`.
```
Is it correct?

Reply:
yes. It is strange to me to deprecate one method, introduce an identical alias but then say "consider using the new version". I think it is more clear to say "please use the new version", or "calls to should be replaced to the new version" or something. Consider implies in my opinion that you may consider it and have a good reason not to, but if we thought there was a good reason not to, we shouldn't deprecate the current name.

Reply:
Totally makes sense to me, thanks!